### ISSUE TITLE
M3.18: Community pipelines

### ISSUE THREAD

**Issue by theathorn** [HumanCellAtlas/dcp#270]
From https://docs.google.com/document/d/14gVAMscd0d8KgWqOO11Y1kSp4ztVavZXA7zrtrhyiW0:

Ensure community pipelines are labeled as such in metadata and this metadatum is surfaced in browser.

**Comment by theathorn** [comment]
@kbergin Can you assign a shepherd from Green team?

**Comment by brianraymor** [comment]
I think that the title for this issue is misleading. It suggests that the issue is about supporting community pipelines but the actual description is related to flagging such pipelines in metadata.

**Comment by brianraymor** [comment]
@kbergin - is this a duplicate of [Method to delineate community-pipeline analyzed data vs HCA-approved pipeline analyzed data](https://github.com/HumanCellAtlas/dcp/issues/56)?

**Comment by kbergin** [comment]
Yes it is @brianraymor . We can keep this one if that's favorable. I'll close the other one. I'm not sure this is the exact implementation/naming we'll end up with, but I am thinking about this proposal currently.

**Comment by kbergin** [comment]
@lauraclarke high level epic about distinguishing between pipeline types. Currently secondary-analysis/406 represents adding it to the metadata, but can be moved to metadata-schema backlog as well! Whatever you prefer. This need will arise in early Q3.

**Comment by kbergin** [comment]
@jkaneria I've tagged you here, the green box related change is an analysis metadata schema update to support distinguishing pipeline output types. The browser and portal are also linked here as they will need to expose that metadata.

**Comment by brianraymor** [comment]
@jkaneria - Based on the [comment](https://github.com/HumanCellAtlas/secondary-analysis/issues/406#issuecomment-514763233) in the child issue:

_We have not yet prioritized it because we haven't yet had a need to install any contributed or non-standard HCA pipelines into the DCP._

This does not sound like a priority for Q3 (and is not one of the Q3 Roadmap objectives). If this is the case, would you remove the Release/Milestone and move to the _Icebox_ pipeline for future consideration? We really want the _Product Backlog_ limited to work scheduled for the current Release.

**Comment by brianraymor** [comment]
@jkaneria - I'm clearing the Release and Milestone and moving this to the Icebox since there's been no update since July 2019. This is also not prioritized for Q4.

**Comment by kbergin** [comment]
This is actively being designed.

**Comment by brianraymor** [comment]
@kbergin - I don't see any status updates in the issue since July - are you referencing the RFC in community review or specific github issues?
